### PR TITLE
Add page layout

### PR DESCRIPTION
**Pull request by thomplinds**
Adds a page layout of cards that each mount the avatar component

image

image

### CODE REVIEW

File: src/views/ListPage.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Avatar } from "../features/avatar";
+import { Card, CardBody, Grid, GridItem, Heading, Text } from "@chakra-ui/react";
+import { contacts } from "../testData";
+
+function ContactCard({ contact }) {
```
Comment (thomplinds) [inline src/views/ListPage.jsx:5]: wasn't sure if the contact card was a component or feature (what's the difference?)

File: src/views/ListPage.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            <CardBody>
+            <Avatar contactId={contact.id} />
+            <Heading>{contact.name}</Heading>
+            <Text>{contact.email}</Text>
+            </CardBody>
+        </Card>
+    );
+}
+
+export default function ListPage() {
```
Comment (dandalf-the-eng) [inline src/views/ListPage.jsx:17]: Named exports only.

File: src/features/avatar/components/avatar-list/avatar-list.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { RobotIcon } from "../../../../components/icons/robot-icon";
 import { UnorderedList, ListItem } from "@chakra-ui/react";
 import { useState } from "react";
 
-const listSize = new Array(10).fill("");
+const listSize = new Array(9).fill("");
```
Comment (dandalf-the-eng) [inline src/features/avatar/components/avatar-list/avatar-list.jsx:5]: Just curious, why'd we drop the count here?

Comment (dandalf-the-eng) [reply src/features/avatar/components/avatar-list/avatar-list.jsx:5]: Ah it's to look nice in edit mode.

**Review by dandalf-the-eng — CHANGES_REQUESTED** [review]
This is awesome, I have one small fix.

File: src/features/avatar/components/avatar-list/avatar-list.jsx (hunk 3 above)
Comment (thomplinds) [reply src/features/avatar/components/avatar-list/avatar-list.jsx:5]: because 9 looks prettier in a grid layout lol

**Pull request merged**